README.md

```diff
@@ -1,80 +1,179 @@
+
+The tool will inspect the source code, locate an appropriate image on DockerHub, create an ImageStream for that image, and then create the right build configuration, deployment configuration and service definition.  
+
+(The -l flag will apply a label of "name=myapp" to all the resources created by new-app, for easy management later.)
+
+#### Create a new app from a template (method 2)
+
+We can also [create new apps using OpenShift template files](https://docs.openshift.com/enterprise/3.0/dev_guide/new_app.html#specifying-a-template). Clone the demo app source code from [GitHub repo](https://github.com/openshift/nodejs-ex) (fork if you like).
+
+        $ git clone https://github.com/openshift/nodejs-ex
         
-That should be it, this form of `new-app` will locate an appropriate image on DockerHub, create an ImageStream for that image, and then create the right build configuration, deployment configuration and service definition.  Next you'll be able to kick off the build, though new-app will kick off a build once all required dependencies are confirmed.  The -l flag will apply a label of "name=myapp" to all the resources created by new-app, for easy management later.
+Looking at the repo, you'll notice two files in the openshift/template directory:
+
+	nodejs-ex
+	├── README.md
+	├── openshift
+	│   └── templates
+	│       ├── nodejs-mongodb.json
+	│       └── nodejs.json
+	├── package.json
+	├── server.js
+	└── views
+	    └── index.html
+	    
+We can create the the new app from the `nodejs.json` template by using the `-f` flag and pointing the tool at a path to the template file:
+
+        $ oc new-app -f /path/to/nodejs.json
+        
+#### Build the app
+
+`oc new-app` will kick off a build once all required dependencies are confirmed. 
+
+Check the status of your new nodejs app with the command:
+
+        $ oc status
 
-Note, you can follow along with the web console (located at https://ip-address:8443/console) to see what new resources have been created and watch the progress of the build and deployment.
+Which should return something like:
 
-#### build ####
+        In project nodejs (nodejs-echo) on server https://10.2.2.2:8443
 
-If the build is not started (you can check by running "oc get builds"), start one and stream the logs with:
+Note the address, as yours may differ. This is the address for the web GUI console. You can follow along with the web console to see what new resources have been created and watch the progress of builds and deployments.
+
+If the build is not yet started (you can check by running `oc get builds`), start one and stream the logs with:
 
         $ oc start-build nodejs-ex --follow
 
-You can alternatively leave off `--follow` and use `oc build-logs nodejs-ex-n` where n is the number of the build to track the output of the build.
+You can alternatively leave off `--follow` and use `oc build-logs nodejs-ex-n` where *n* is the number of the build to track the output of the build.
```

Comment:
build-logs has been deprecated, can you switch this to "oc logs build/nodejs-ex-n" ?